FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -55,11 +61,17 @@ class IssuesViewController : UITableViewController {
         Answers.logCustomEvent(withName:"Screen: Issues List")
         
         navigationController?.setNavigationBarHidden(true, animated: false)
-        loadIssues()
-        
-        tableView.estimatedRowHeight = 75
-        tableView.rowHeight = UITableViewAutomaticDimension
-        
+
+        // Refetch data only if we don't have anything to display.
+        if viewModel.categories.count == 0 {
+            loadIssues()
+        }
+
+        // 'Whats important' header gets shown only when we are showing
+        // active issues only.
+        if !shouldShowAllIssues {
+            tableView.tableHeaderView = headerWithTitle(title: R.string.localizable.whatsImportantTitle())
+        }
```

Comment:
Thinking this would be simpler to show as a section header, so we wouldn't need this little bit of complexity...

Reply:
Will do this change. Thanks.